### ISSUE TITLE
Hangs on  get  https://www.symantec.com/content/en/us/enterprise/verisign/roots/VeriSign-Class%203-Public-Primary-Certification-Authority-G5.pem ->  200

### ISSUE THREAD

**Issue by olskar** [weweave/landroid-bridge#4]
Tried running this both from docker and building from source, seems to work but hangs on the following message:

get  https://www.symantec.com/content/en/us/enterprise/verisign/roots/VeriSign-Class%203-Public-Primary-Certification-Authority-G5.pem ->  200

**Comment by virtualzone** [comment]
I've just pushed a new version to GitHub and Docker Hub which contains a newer version of the iobroker.landroid-s library. This should fix the problem. Can you give it a try?

**Comment by olskar** [comment]
@virtualzone Gives me below message now

`Loading config from /home/pi/mower/landroid-bridge/config.json
module.js:549
    throw err;
    ^

Error: Cannot find module 'iobroker.landroid-s/lib/landroid-cloud-2'
    at Function.Module._resolveFilename (module.js:547:15)
    at Function.Module._load (module.js:474:25)
    at Module.require (module.js:596:17)
    at require (internal/module.js:11:18)
    at Object.<anonymous> (/home/pi/mower/landroid-bridge/dist/LandroidS.js:3:23)
    at Module._compile (module.js:652:30)
    at Object.Module._extensions..js (module.js:663:10)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
`

**Comment by virtualzone** [comment]
@olskar Pushed another update, can you give it a try?

**Comment by olskar** [comment]
Now  I get this, after that nothing:

`Loading config from /home/pi/mower/landroid-bridge/config.json
Dev mode = undefined
Starting Landroid Bridge...
Setting port to 3000...
Connecting to MQTT Broker...
Server ready
Successfully connected to MQTT Broker!`

**Comment by olskar** [comment]
curl -X GET http://localhost:3000/landroid-s/status gives me:

`
{"message":"Internal Server Error"}`

**Comment by olskar** [comment]
Works now!

**Comment by riker09** [comment]
I'm currently getting the same error when I try to use the bridge. I had it working with the credentials from a friend who happens to own the same model as I do. Since all I did was changing username and password I'm suspecting this has to do something with it. The error is

```
Unhandled exception: TypeError: Cannot read property 'mac_address' of undefined
```

I was using the app with a Google login before, so I did not have any password set. I tried setting one but it looks like I f***** up my account/landroid somehow. I'm waiting for the support to unpair my landroid so I can pair it with my password protected account and I will report back here if that fixes the issue.

Why am I writing all this? From the looks of it I believe that @olskar had a similar issue.

On a sidenote I believe the error message send by the bridge could be improved in such a case. I will work something out once I can get it back to a working state.
